cmake/bazel.cmake

```diff
@@ -52,7 +52,7 @@ add_custom_target(helps COMMAND ./make_help WORKING_DIRECTORY ${BAZEL_SOURCE_DIR
 
 macro(_build_target func_tag)
   set(_sources ${ARGN})
-  list(FILTER _sources EXCLUDE REGEX ".proto$")
+  list(FILTER _sources EXCLUDE REGEX "\\.proto$")
```

Comment:
Not sure what's the meaning of `\\`.

It that similar to `[a-zA-z0-9]+[.]proto$`?

Reply:
Regular Expressions http://www.mpi.nl/corpus/html/trova/ch01s04.html

Reply:
https://cmake.org/pipermail/cmake/2004-May/005136.html